scripts/core/superdesk-api.d.ts

```diff
@@ -276,6 +278,15 @@ declare module 'superdesk-api' {
         };
     }
 
+    export interface IPublishedArticle extends IArticle {
```

Comment:
As I've written in `SDESK-4687` it would be good to unify the articles so the structure is the same whether it's published or not. Do you know why `archive_item` and `item_id` is used? Wouldn't `status: 'published;` work while maintaining the same structure?

Reply:
item_id is there because there are multiple versions of the same item in published collection, so there is another id for that. why there is archive_item I'm not sure exactly

Reply:
Items that aren't published also have different versions you can revert back to. No matter the state of an article, it could probably use the same versioning management.

Reply:
yep but there is only 1 version visible at all times, while with published we display them all together

Reply:
I don't know how it's relevant, but it sounds like just another requirement to keep in mind rather than a limitation that would make it impossible to unify `published` and `archive` into a single entity. What do you think?

Reply:
I'm sure it's possible